suitescript/suitescript.d.ts

```diff
@@ -2966,7 +3017,7 @@ declare interface nlobjRecord {
      * @since 2009.2
      * @param group
      */
-    getLineItemCount(group:string): void;
+    getLineItemCount(group:string): any;
```

Comment:
It is with my 6yrs experience with the API that I know this function return
string as well as in, but also null and undefined.

You'll notice the bulk of my updates are correcting these function
signatures, not really too many ADDs